pymc/distributions/continuous.py

```diff
@@ -1461,8 +1461,8 @@ class Exponential(PositiveContinuous):
 
     Parameters
     ----------
-    lam: float
-        Rate or inverse scale (lam > 0)
+    lam : float
+        Rate or inverse scale (`lam` > 0)
```

Comment:
```suggestion
        Rate or inverse scale (``lam`` > 0)
```

dosctrings are in rST format which needs two backticks for inline code formatting.